calico_containers/integrations/kubernetes/tests/kube_plugin_test.py

```diff
@@ -0,0 +1,414 @@
+# Copyright 2015 Metaswitch Networks
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import sys
+if sys.version_info.major == 2:
+    import __builtin__ as builtins
+else:
+    import builtins
+import json
+import unittest
+import socket
+from mock import patch, Mock, MagicMock, mock_open, call
+from subprocess import CalledProcessError
+from calico_containers.integrations.kubernetes import calico_kubernetes
+from pycalico.datastore import IF_PREFIX
+
+class NetworkPluginTest(unittest.TestCase):
+
+    def setUp(self):
+        # Mock out sh so it doesn't fail when trying to find the
+        # calicoctl binary (which may not exist)
+        with patch('calico_containers.integrations.kubernetes.'
+                   'calico_kubernetes.sh') as m_sh:
+            self.plugin = calico_kubernetes.NetworkPlugin()
+
+    def test_create(self):
+        with patch.object(self.plugin, '_configure_interface',
+                          autospec=True) as m_interface, \
+                patch.object(self.plugin, '_configure_profile',
+                             autospec=True) as m_prof:
+            # Set up mock objects
+            m_interface.return_value = 'endpt_id'
+
+            # Set up args
+            args = ['script name','setup','','pod-name','dockerId']
```

Comment:
Proper list formatting includes a space after each item in the list.

Like so:
```
args = ['script name', 'setup', '', 'pod-name', 'dockerId']
```

I think we should figure out what args[2] is supposed to be and put something there.  Talk to me if you're unsure how to find this out.